### DISCUSSION TITLE
why there is no gini importance in the function rfsrc?

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#241] [General]
![image](https://user-images.githubusercontent.com/5112397/166932814-9b6dd4dd-10b8-432f-9795-b277836b130c.png)
why there is no gini importance in the function rfsrc? Just because it is biased?

**Comment by ishwaran** [comment]
The option you highlighted selects the type of importance.  Since this is not family specific (i.e. works for survival, regression, etc) you can understand this is not where you would set an option to choose performance error.  To set the performance metric you must use `perf.type.`  However, currently gini is not implemented.

See **Specifying the Type of Predicted Value, Variable Importance and Error Rate**

https://www.randomforestsrc.org/articles/getstarted.html
